manifests/tftp.pp

```diff
@@ -13,8 +13,7 @@
     recurse => true;
   }
 
-  foreman_proxy::tftp::sync_file{$foreman_proxy::tftp_syslinux_files:
```

Comment:
Certainly we should print a warning, but I'm not sure about syncing them as well. In a pure puppet world I would agree but with the installer and old answer files I'm afraid about duplicate syncs. If a user upgraded their debian as well it would lead to errors. OTOH that's a bit of an edge case.